{{ cookiecutter.folder_name }}/.travis.yml

```diff
@@ -55,3 +99,14 @@ script:
 after_success:
   - coverage report -m
   - codecov
+
```

Comment:
This will fail if you don't set these environment variables in Travis. Maybe this is desirable.